### DISCUSSION TITLE
Update Playbook Sigma rules

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#7755] [Unsupported Versions] [answered]
Hello all,

If you compare the sigma rules in the SO installation with the sigma rules in the [gitlab repo](https://github.com/SigmaHQ/sigma/tree/master/rules/windows), the rules is SO haven't changed since end 2019.

An example:
Gitlab repo Autorun Keys Modification id: 17f878b8-9968-4578-b814-c4217fc5768c  modified: 2021/12/05
SO repo Autorun Keys Modification id: 17f878b8-9968-4578-b814-c4217fc5768c modified: 2019/11/10

Even if you use the command so-playbook-ruleupdate you still have the old rules.

Is there a possibility to pull the latest sigma rules directly from the gitlab repo, because several rules have been adjusted in the meantime?

Regards
Bart

**Comment by defensivedepth** [comment, marked as answer]
We are working on overhauling this.

For now, I have created an issue to track adding the ability to override the community sigma repo to one of your choice.
